__tests__/app.test.js

```diff
@@ -328,3 +328,20 @@ describe('DELETE /api/comments/:comment_id', () => {
       });
   });
 });
+
+describe('GET /api/users', () => {
+  test('200: should return an array of user objects with username, name, and avatar_url', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.users)).toBe(true);
+        expect(body.users.length).toBeGreaterThan(0);
+        body.users.forEach((user) => {
+          expect(user).toHaveProperty('username');
+          expect(user).toHaveProperty('name');
+          expect(user).toHaveProperty('avatar_url');
```

Comment:
What about the values? Is there a matcher we could use to test for these properties ___and___ test that the values are the correct data types?